### PR TITLE
Feature/python3.8 support

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Config file for automatic testing at travis-ci.org
-sudo: false
+sudo: required  # for 3.7, see https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905
```
Comment (hroncok) [inline .travis.yml]: This might no longer be necessary.

File: lib/astunparse/unparser.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ class Unparser:
     output source code for the abstract syntax; original formatting
     is disregarded. """
 
-    def __init__(self, tree, file=sys.stdout):
+    def __init__(self, tree, file = sys.stdout):
```
Comment (hroncok) [inline lib/astunparse/unparser.py]:
```suggestion
    def __init__(self, tree, file=sys.stdout):
```

Comment (serge-sans-paille) [reply lib/astunparse/unparser.py]: Just keeping in sync with original cpython file there

**Comment by serge-sans-paille** [comment]
@simonpercivall this looks fine now

**Comment by serge-sans-paille** [comment]
up :-)

**Comment by simonpercivall** [comment]
Thank you! and sorry for the time it took.

**Pull request merged**